docs/conversion/sakai_20_oracle_conversion.sql

```diff
@@ -86,3 +86,8 @@ ALTER TABLE SAM_PUBLISHEDACCESSCONTROL_T ADD FEEDBACKSCORETHRESHOLD FLOAT(126);
 
 -- SAK-42498
 ALTER TABLE BULLHORN_ALERTS DROP COLUMN ALERT_TYPE;
+
+-- START SAK-41812
+ALTER TABLE sakai_person_t ADD phonetic_pronunciation longtext default null;
```

Comment:
`longtext` is not a valid Oracle data type as far as I'm aware.

Reply:
Exactly, Hibernate is creating a clob instead:
![image](https://user-images.githubusercontent.com/8653754/76324216-fa06bd80-62e5-11ea-9f62-dd6d557e0fe0.png)